src/plugins/discover/public/application/main/components/sidebar/discover_sidebar_responsive.test.tsx

```diff
@@ -199,19 +202,30 @@ async function mountComponent(
   mockedServices.data.query.getState = jest.fn().mockImplementation(() => appState.getState());
 
   await act(async () => {
-    comp = await mountWithIntl(
+    const SidebarWrapper = () => {
+      const [api, setApi] = useState<UnifiedFieldListSidebarContainerApi | null>(null);
+      return (
+        <DiscoverSidebarResponsive
+          {...props}
+          unifiedFieldListSidebarContainerApi={api}
+          setUnifiedFieldListSidebarContainerApi={setApi}
+        />
+      );
+    };
+
+    comp = mountWithIntl(
       <KibanaContextProvider services={mockedServices}>
         <DiscoverAppStateProvider value={appState}>
-          <DiscoverSidebarResponsive {...props} />
+          <SidebarWrapper />
         </DiscoverAppStateProvider>
       </KibanaContextProvider>
     );
     // wait for lazy modules
     await new Promise((resolve) => setTimeout(resolve, 0));
-    await comp.update();
+    comp.update();
```

Comment:
I removed the `await`s while troubleshooting test failures, but figured I'd leave them gone since the functions aren't async.